src/docs/resources/videos.md

```diff
@@ -95,6 +95,8 @@ millions of downloads.
 Learn how to build Flutter apps from this free Udacity course:
```

Comment:
Can you modify this line to:
Learn how to build Flutter apps with these free video courses.